http/rest-client-reactive/src/main/resources/httpVersion.properties

```diff
@@ -0,0 +1,3 @@
+// general REST client setting
```

Comment:
1. properties comments shouldn't created with `//` whatever your framework works. If our framework really requires this, please open issue in Framework project
2. I don't think this comment is valuable. If you follow this pattern, you will have to comment out everything. One of million examples - `quarkus.http.ssl.certificate.key-store-file` in the file right above, is it better known to readers than following properties?
3. The comment doesn't seem true, this is client setting specific to concrete named clients?

I'm not against comments at all, but this one particular strikes me. Thouths @jedla97 ?

Reply:
Removed the comment. I had there part for server side settings so that make sense for me. But now agree that it's not needed.